doc/nrf/links.txt

```diff
@@ -612,6 +612,7 @@
 
 .. _`ST25R3911B chip documentation`: https://www.st.com/resource/en/datasheet/st25r3911b.pdf
 
+.. _`Connectivity Standards Alliance`: https://csa-iot.org
 .. _`Zigbee Alliance`: https://zigbeealliance.org
```

Comment:
```suggestion
.. _`Zigbee Alliance`:
.. _`Connectivity Standards Alliance`: https://csa-iot.org
```
This will use the same page for both links. Zigbee Alliance is referenced in one of the old release notes, so I don't think we need a specific link for the alliance anymore, and this redirection should be enough.